README.md

```diff
@@ -27,19 +27,18 @@ If you want a quick verification that everything has installed correctly, the fo
 <script type="text/javascript">
 var data = fc.data.random.financial()(50);
```

Comment:
N.B. With the current release, this won't work, as it's still `fc.dataGenerator()`. When 4897b48baf942255e6e3bea52060efa0486bb140 is included in the next release, `fc.data.random.financial()` will be correct.